R/FilteredDataset.R

```diff
@@ -231,6 +231,20 @@ FilteredDataset <- R6::R6Class( # nolint
             column(
               width = 4,
               tagList(
+                actionLink(
+                  ns("show_new_filter_ui"),
+                  label = "",
+                  icon = icon("plus", lib = "font-awesome"),
+                  class = "remove pull-right"
+                ),
+                shinyjs::hidden(
+                  actionLink(
+                    ns("hide_new_filter_ui"),
+                    label = "",
+                    icon = icon("minus", lib = "font-awesome"),
+                    class = "remove pull-right"
+                  )
+                ),
```

Comment:
I think we can have a single button which toggles `add` panel with consistent naming.

button: `show_hide_add_panel`
panel: `add_panel` or just `add`